ocean_lib/services/service.py

```diff
@@ -272,3 +301,79 @@ def update_compute_values(
         ] = trusted_algo_publishers
         self.compute_values["allowNetworkAccess"] = allow_network_access
         self.compute_values["allowRawAlgorithm"] = allow_raw_algorithm
+
+
+class ConsumerParameters:
+    def __init__(
+        self,
+        name: str,
+        type: str,
+        label: str,
+        required: bool,
+        default: str,
+        description: str,
+        options: Optional[List[str]] = None,
+    ) -> None:
+
+        fn_args = locals().copy()
+        for attr_name in ConsumerParameters.required_attrs():
+            setattr(self, attr_name, fn_args[attr_name])
+
+        if options is not None and not isinstance(options, list):
+            raise TypeError("Options should be a list")
+
+        self.options = options
+
+    @classmethod
+    def from_dict(
+        cls, consumer_parameters_dict: Dict[str, Any]
+    ) -> "ConsumerParameters":
+        """Create a ConsumerParameters object from a JSON string."""
+        sd = copy.deepcopy(consumer_parameters_dict)
```

Comment:
Instead of `sd` which I assume was copied from `Service` where it's shorthand for "service dict", could you please change this to `cpd`, shorthand for "consumer parameters dict"?